examples/_code-sandbox.js

```diff
@@ -0,0 +1,177 @@
+    const txtDataFile = await responseFile.text();
+
+    additionalJsFiles[filePath.replace('./', '').replace('rawjs', '')] = {content: txtDataFile};
+  }
+
+  initCodeSandboxButton({indexJsContent, additionalJsFiles, resourcesFiles});
+}
+
+function initCodeSandboxButton(options) {
+  const {indexJsContent, additionalJsFiles, resourcesFiles} = options;
+
+  let indexHtmlContent = '';
+  const button = document.getElementById('sandbox-button');
+  const form = document.querySelector('#sandbox-form');
+
+  if (!button || !form) {
+    return;
+  }
+
+  const divExampleCodeSource = document.createElement('div');
+  divExampleCodeSource.innerHTML = document.getElementById('example-html-source').innerHTML;
+  divExampleCodeSource.querySelectorAll('.clear-map-sandbox').forEach(map => map.innerHTML = '');
+  indexHtmlContent = divExampleCodeSource.innerHTML;
+
+  const indexHtml = `
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Ol-Cesium example in sandbox</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="node_modules/ol/ol.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <script src="https://cesium.com/downloads/cesiumjs/releases/1.116/Build/Cesium/Cesium.js"></script>
+    <style>
+      .ol-popup {
+        position: absolute;
+        background-color: white;
+        -webkit-filter: drop-shadow(0 1px 4px rgba(0,0,0,0.2));
+        filter: drop-shadow(0 1px 4px rgba(0,0,0,0.2));
+        padding: 15px;
+        border-radius: 10px;
+        border: 1px solid #cccccc;
+        bottom: 12px;
+        left: -50px;
+        min-width: 280px;
+      }
+      .ol-popup:after, .ol-popup:before {
+        top: 100%;
+        border: solid transparent;
+        content: " ";
+        height: 0;
+        width: 0;
+        position: absolute;
+        pointer-events: none;
+      }
+      .ol-popup:after {
+        border-top-color: white;
+        border-width: 10px;
+        left: 48px;
+        margin-left: -10px;
+      }
+      .ol-popup:before {
+        border-top-color: #cccccc;
+        border-width: 11px;
+        left: 48px;
+        margin-left: -11px;
+      }
+      .ol-popup-closer {
+        text-decoration: none;
+        position: absolute;
+        top: 2px;
+        right: 8px;
+      }
+      .ol-popup-closer:after {
+        content: "✖";
+      }
+      .popover-content {
+        min-width: 180px;
+      }
+      code {
+        padding: 2px 4px;
+        font-size: 90%;
+        color: #c7254e;
+        background-color: #f9f2f4;
+        border-radius: 4px;
+      }
+    </style>
+  </head>
+  <body>
+  ${indexHtmlContent}
+  </body>
+  <script type="module" src="/index.js"></script>  
+  <script src="https://code.jquery.com/jquery-2.2.3.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+</html>`;
+  const parameters = {
+    template: 'parcel',
```

Comment:
vite?

Reply:
`template: 'vite',` breaks the codesandbox loading => I have removed the full template line